frontend/src/wrapper/SideNavigation.tsx

```diff
@@ -151,6 +152,15 @@ export default function SideNavigation({
               icon={<ContactSupportIcon />}
             />
             <Divider />
+            <NavMenuItem
```

Comment:
Could this be put just below the 'API' menu item?  I like the similarities between the 'API' and 'Python Client' docs.